app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title
+    base_title = t(".helper")
+    if page_title.blank?
+    base_title
+    else
+    page_title + " | " + base_title
+    end
```

Comment:
Edit lại format indent
```suggestion
    if page_title.blank?
      base_title
    else
      page_title + " | " + base_title
    end
```
hoặc có thể dùng toán tử 3 ngôi để check đk cho gọn trên 1 line
```suggestion
    page_title.blank? ? base_title : page_title + " | " + base_title
```